apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -3847,13 +3821,13 @@ sc_ws_withdraw_(Config, Origin) when Origin =:= initiator
                                                               error]),
     ok = ?WS:register_test_for_channel_events(AckConnPid, [sign, info, on_chain_tx,
                                                            error]),
-    ws_send(SenderConnPid, <<"deposit">>, #{amount => <<"2">>}, Config),
+    ws_send_tagged(SenderConnPid, <<"channels.deposit">>, #{amount => <<"2">>}, Config),
```

Comment:
There is conflict with master https://github.com/aeternity/aeternity/commit/d80e4a2ed5f75d4746f020c9b8e99c105f607c05
because of this line.